generators/app/steps/CustomizableSteps/index.js

```diff
@@ -1,15 +1,14 @@
 const installDependencies = require('../../tasks/installDependencies');
 
-const copyAllFiles = require('./copyFiles');
+const copyAllFiles = require('./copyAllFiles');
 const { DEPENDENCIES, DEV_DEPENDENCIES } = require('./constants');
-const { CUSTOMIZED_PROMPTS } = require('./prompts');
 
 const CustomizableGenerator = {
   prompting() {
-    return this.prompt(CUSTOMIZED_PROMPTS);
+    return Promise.resolve();
```

Comment:
You removed the prompts? Why?

Reply:
They were optional components that we don't want anymore. I don't think we have a case for prompts right now. We can restore it if needed.